modules/blocks/pages/build-basic-block.adoc

```diff
@@ -4,7 +4,7 @@
 
 == Delimited blocks
 
-Block content can be styled by enclosing it in a set of delimiters.
+Block content can be styled by enclosing it in a pair of delimiters (aka fences).
```

Comment:
I don't think the "aka fences" adds anything to help, especially since we seem to be removing/limiting mentions of the fenced source block which I know isn't related to this usage but it could muddy the waters so to speak.

Reply:
I see your point. The only reason I suggested it is because I'm not sure if a writer will understand what delimiter lines are. But maybe thinking of them as fences is also assumes familiarity with an existing mental model. Probably best, then, to let the examples do the talking.